### ISSUE TITLE
TypeError: DiscordUnix._get_pipe_path() takes 0 positional arguments but 1 was given

### ISSUE THREAD

**Issue by keisanng** [Senophyx/Discord-RPC#11]
```bash
  File "project_path/src/__main__.py", line 15, in main
    RPC = discordrpc.RPC.set_id(app_id=CLIENT_ID)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "project_path/.venv/lib/python3.11/site-packages/discordrpc/presence.py", line 33, in set_id
    return DiscordUnix(app_id)
           ^^^^^^^^^^^^^^^^^^^
  File "project_path/.venv/lib/python3.11/site-packages/discordrpc/presence.py", line 20, in __init__
    self._connect()
  File "project_path/.venv/lib/python3.11/site-packages/discordrpc/presence.py", line 218, in _connect
    pipe_path = self._get_pipe_path()
                ^^^^^^^^^^^^^^^^^^^^^
TypeError: DiscordUnix._get_pipe_path() takes 0 positional arguments but 1 was given
```

**Comment by LyQuid12** [comment]
Try different version first, check whether the error is still the same

**Comment by pazkero** [comment]
Same error, using example handbook, discordrpc version 4.0 Python 3.10 Ubuntu 22.04

Python 3.10 is listed as "supported" in https://pypi.org/project/discord-rpc/ from where I used the code.

**NOT WHEEL**.

**Comment by Senophyx** [comment]
> Same error, using example handbook, discordrpc version 4.0 Python 3.10 Ubuntu 22.04

Linux-based OS has not been fully tested because I only tested it on a virtual machine and only some versions have been tested.

> Python 3.10 is listed as "supported" in https://pypi.org/project/discord-rpc/ from where I used the code.

What I meant before was try a different version of Discord-RPC (maybe this is my fault for not providing a complete explanation). I'm sorry.

> Proposed patch in #13 

Thank you so much for your contributions. I will test the patch you provided as soon as possible. While waiting for the latest version update, you can try installing Discord-RPC using the repo that you have forked which already contains your patch and report to me if all functions (such as in the examples folder) run smoothly.

**Comment by pazkero** [comment]
> > Proposed patch in #13
> 
> Thank you so much for your contributions. I will test the patch you provided as soon as possible. While waiting for the latest version update, you can try installing Discord-RPC using the repo that you have forked which already contains your patch and report to me if all functions (such as in the examples folder) run smoothly.

![image](https://github.com/Senophyx/Discord-RPC/assets/8108358/76cb8214-10f1-4cbb-9118-bc0f06b95900)

Yes, I tested it. And yes, it works.

**Comment by Senophyx** [comment]
Very great! Do all the functions work?

Also, Most likely I will release the latest version next week because I'm still busy right now.

**Comment by Senophyx** [comment]
Fixed on #13.
